hazelcast/src/main/java/com/hazelcast/config/CacheConfig.java

```diff
@@ -71,6 +71,7 @@
     private WanReplicationRef wanReplicationRef;
     private List<CachePartitionLostListenerConfig> partitionLostListenerConfigs;
     private String quorumName;
+    private String mergePolicy = CacheSimpleConfig.DEFAULT_CACHE_MERGE_POLICY;
```

Comment:
I have removed `quorumName` from the `toString` method since I think it is not necessary giving information at there.